src/main/java/io/github/amerebagatelle/fabricskyboxes/SkyboxManager.java

```diff
@@ -152,7 +169,12 @@ public void setEnabled(boolean enabled) {
         this.enabled = enabled;
     }
 
-    public static SkyboxManager getInstance() {
-        return INSTANCE;
+    public Skybox getCurrentSkybox() {
+        return this.currentSkybox;
+    }
+
+    @Override
+    public int getApiVersion() {
```

Comment:
Not sure how this is useful?  If the API is outdated, the API is outdated.  It's not like users aren't going to notice code crashing, and semver should be bumped whenever the API is updated anyways.  Also it's just not that helpful if we don't support old API versions anyways (which I don't think we should)

Reply:
It allows other mods to check the version of the API before interacting with it so it does not cause crashes.

Reply:
That's why we have SemVer though, and mods can just as easily get the version of the mod itself through the Fabric/Quilt loader API provisions if worried about crashes.

Reply:
The mod version shouldn't be used as the API version because the API feature set may be the same across multiple versions.